src/main/java/org/jboss/tools/intellij/openshift/utils/odo/JSonParser.java

```diff
@@ -123,41 +126,36 @@ public ComponentInfo parseComponentInfo(ComponentKind kind) {
 
     public ComponentInfo parseDescribeComponentInfo(ComponentKind kind) {
         ComponentInfo.Builder builder = new ComponentInfo.Builder().withComponentKind(kind);
-        if (root.has(DEVFILE_DATA_FIELD) && root.get(DEVFILE_DATA_FIELD).has(DEVFILE_FIELD) &&
-                root.get(DEVFILE_DATA_FIELD).get(DEVFILE_FIELD).has(METADATA_FIELD) &&
-                root.get(DEVFILE_DATA_FIELD).get(DEVFILE_FIELD).get(METADATA_FIELD).has(PROJECT_TYPE_FIELD)) {
+        if (root.has(DEVFILE_DATA_FIELD) && root.get(DEVFILE_DATA_FIELD).has(DEVFILE_FIELD) && root.get(DEVFILE_DATA_FIELD).get(DEVFILE_FIELD).has(METADATA_FIELD) && root.get(DEVFILE_DATA_FIELD).get(DEVFILE_FIELD).get(METADATA_FIELD).has(PROJECT_TYPE_FIELD)) {
```

Comment:
nitpicking: liked the old formatting where each &&-condition was on a separate line and thus was easier to be read